pyvista/core/filters/data_set.py

```diff
@@ -1239,7 +1239,7 @@ def contour(self, isosurfaces=10, scalars=None, compute_normals=False,
             Number of isosurfaces to compute across valid data range or a
             sequence of float values to explicitly use as the isosurfaces.
 
-        scalars : str, optional
+        scalars : str, iterable, optional
```

Comment:
Is this true?  It looks like it takes an ndarray.

Reply:
Fixed in 71d4ef69f40d9afc539df7e1e9427a80174c13f4.